books/free-programming-books-he.md

```diff
@@ -0,0 +1,36 @@
+### Index
+
+* [C#](#cs)
+* [Assembly](#assembly)
```

Comment:
- We should preserve order in TOC too :wink:
- Like in other listings use `c-sharp` as anchor alias

```suggestion
* [Assembly](#assembly)
* [C#](#c-sharp)
```